src/bin/cli.ts

```diff
@@ -28,6 +35,11 @@ const main = async () => {
     outdir: path.resolve(process.cwd(), outdir),
     node,
     cwd: process.cwd(),
+    esbuild: includeAwsSdk
```

Comment:
I've been using esbuild and blamda in projects together, I would prefer to pass this down as a variable, instead of overriding the external array.

Reply:
You made me catch a bug just now.  Here is how I'm using it in `patient-service`
https://github.com/lifeomic/patient-service/blob/19fc180ef3be916ecddf49af0b507fd927cb7912/build.ts

Reply:
so something like this @DavidTanner?

```diff --git a/src/bundle.ts b/src/bundle.ts
index 55a15cc..e97f175 100644
--- a/src/bundle.ts
+++ b/src/bundle.ts
@@ -61,7 +61,11 @@ export const bundle = async ({
        *
        * https://aws.amazon.com/blogs/compute/node-js-18-x-runtime-now-available-in-aws-lambda/
        */
-      external: nodeVersion >= 18 ? ['@aws-sdk/*'] : ['aws-sdk'],
+      external: includeAwsSdk
+        ? []
+        : nodeVersion >= 18
+        ? ['@aws-sdk/*']
+        : ['aws-sdk'],
       /**
        * As of v0.14.44, esbuild by default prefers .ts over .js files.
        *
(END)
```

Reply:
Yes, then this would be 
```suggestion
    includeAwsSdk,
```